### PR TITLE
Footer #1166

### PR DESCRIPTION
**Pull request by phpgeekfr**

### CODE REVIEW

File: libs/ui/src/lib/toolbar/left-menu/left-menu.component.html [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <div class="sidenav_container">
-  <div fxLayout="column" fxLayoutGap="10px" style="height: 100%;">
-    <mat-toolbar class="mat-elevation-z2 left-sidenav-header" fxLayout="row-reverse"
-      fxLayoutAlign="space-between center">
+  <div style="height: 100%;">
```
Comment (Hugo-Hontans) [inline libs/ui/src/lib/toolbar/left-menu/left-menu.component.html]: Plz do this in the scss file

File: styles/style.scss [hunk 2]
```diff
@@ -101,4 +101,26 @@ main {
...
+
+.dark-theme {
+  footer {
+    background: mat-color($bf-dark-background, app-bar);
+    border-top: 1px solid mat-color($bf-dark-background, border);
+
+    a {
+      color: mat-color($bf-dark-background, text);
+    }
+  }
```
Comment (Hugo-Hontans) [inline styles/style.scss]: Do you see with Clélia to manage it ?
I'm not sure that it is well to change a tag color

File: libs/ui/src/lib/toolbar/footer/footer.component.scss [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      li {
+        margin-right: 10px;
+        margin-left: 10px;
+        font-weight: 500;
+
+        a, a:hover, a:visited {
+          color: inherit;
+          font-size: 12px;
+          line-height: 15px;
+          text-decoration: none;
```
Comment (Hugo-Hontans) [inline libs/ui/src/lib/toolbar/footer/footer.component.scss]: Maybe we have to do this in global style

File: libs/material/src/lib/delivery/pages/delivery-add-choose-starter/delivery-add-choose-starter.component.html [hunk 4]
```diff
@@ -1,9 +1,9 @@
-<main>
+<div>
```
Comment (Hugo-Hontans) [inline libs/material/src/lib/delivery/pages/delivery-add-choose-starter/delivery-add-choose-starter.component.html]: Do you really need this div ?

File: libs/material/src/lib/delivery/pages/delivery-add-find-movie/delivery-add-find-movie.component.html [hunk 5]
```diff
@@ -1,9 +1,9 @@
-<main>
+<div>
```
Comment (Hugo-Hontans) [inline libs/material/src/lib/delivery/pages/delivery-add-find-movie/delivery-add-find-movie.component.html]: same

File: libs/ui/src/lib/toolbar/footer/footer.component.scss (hunk 3 above)
Comment (phpgeekfr) [reply libs/ui/src/lib/toolbar/footer/footer.component.scss]: I would say no because this is specific to footer

File: styles/style.scss (hunk 2 above)
Comment (phpgeekfr) [reply styles/style.scss]: moved to a specific scss file

**Pull request merged**